src/content/it/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count.md

```diff
@@ -0,0 +1,145 @@
+}
+```
+
+Questo segnala al browser l'avviso che i cambiamenti sono in arrivo e, a seconda
+di ciò che si prevede di cambiare, il browser può potenzialmente fare
+predisposizioni, come creare livelli di composizione.
+
+## Gestisci i livelli ed evita troppi livelli
+
+È molto allettante sapere che i livelli spesso aiutano le prestazioni per
+promuovere tutti gli elementi della pagina con qualcosa di simile al seguente:
+
+```
+* {
+  will-change: transform;
+  transform: translateZ(0);
+}
+```
+
+Il che è un modo indiretto per dire che vorresti promuovere ogni singolo
+elemento della pagina. Il problema qui è che ogni livello creato richiede
+memoria e risorse per la gestione che non sono gratuite. In effetti, su
+dispositivi con memoria limitata l'impatto sulle prestazioni può superare di
+gran lunga qualsiasi vantaggio derivante dalla creazione di un livello. Le
+texture di ogni livello devono essere caricate nella GPU quindi ci sono
+ulteriori limiti in termini di larghezza di banda tra CPU e GPU e memoria
+disponibile per le texture nella GPU.
+
+Warning: non promuovere elementi non necessari.
+
+## Utilizza Chrome DevTools per comprendere i livelli nella tua app
+
+<div class="attempt-right">
+  <figure>
+<img
+src="images/stick-to-compositor-only-properties-and-manage-layer-count/paint-profiler.jpg"
+alt="The toggle for the paint profiler in Chrome DevTools.">
+  </figure>
+</div>
+
+Per avere una comprensione dei livelli nell'applicazione e perché un elemento ha
+un determinato livello devi abilitare il Paint profiler nella timeline degli
+Strumenti per Sviluppatori di Chrome:
+
+<div style="clear:both;"></div>
+
+Con questo abilitato dovresti fare una registrazione. Al termine della
+registrazione potrai fare clic sui singoli frame che si trovano tra le barre dei
+frame per secondo ed i dettagli:
+
+<img
+src="images/stick-to-compositor-only-properties-and-manage-layer-count/frame-of-interest.jpg"
+alt="A frame the developer is interested in profiling.">
+
+Cliccando su questo ti verrà fornita una nuova opzione nei dettagli: il layer
+tab.
+
+<img
+src="images/stick-to-compositor-only-properties-and-manage-layer-count/layer-tab.jpg"
+alt="The layer tab button in Chrome DevTools.">
+
+Questa opzione mostrerà una nuova vista che ti consente di eseguire una
+panoramica, una scansione e uno zoom su tutti i livelli durante quel fotogramma,
+insieme ai motivi per cui ogni livello è stato creato.
+
+<img
+src="images/stick-to-compositor-only-properties-and-manage-layer-count/layer-view.jpg"
+alt="The layer view in Chrome DevTools.">
+
+Usando questa vista puoi tenere traccia del numero di livelli che hai. Se stai
+spendendo molto tempo nel compositing durante azioni critiche per le prestazioni
+come lo scorrimento o le transizioni (dovresti mirare a circa **4-5 ms** ). Puoi
+usare queste informazioni per vedere quanti livelli sono in uso e perché sono
+stati creati e da lì gestire il conteggio dei livelli della tua app.
```

Comment:
Per our [style guide](https://developers.google.com/web/resources/translations#credit), please put the translated by at the bottom of the article.